raiden/transfer/mediated_transfer/initiator_manager.py

```diff
@@ -383,6 +390,100 @@ def handle_transferreroute(
     return TransitionResult(payment_state, events)
 
 
+def handle_transfer_reroute_light(
+    payment_state: InitiatorPaymentState,
+    state_change: ActionTransferRerouteLight,
+    channel_identifiers_to_channels: ChannelMap,
+    storage: SerializedSQLiteStorage
+) -> TransitionResult[InitiatorPaymentState]:
+
+    print("raiden/transfer/mediated_transfer/initiator_manager.py:396 running handle_transfer_reroute_light")
```

Comment:
you probably didn't mean to leave this one here

Reply:
yup removing it